src/mcp/client/streamable_http.py

```diff
@@ -359,6 +367,20 @@ async def _send_session_terminated_error(
         session_message = SessionMessage(JSONRPCMessage(jsonrpc_error))
         await read_stream_writer.send(session_message)
 
+    async def _send_bad_request_error(
+        self,
+        read_stream_writer: StreamWriter,
+        request_id: RequestId,
+    ) -> None:
+        """Send a bad request error response."""
+        jsonrpc_error = JSONRPCError(
+            jsonrpc="2.0",
+            id=request_id,
+            error=ErrorData(code=32600, message="Bad request"),
```

Comment:
There is an INVALID_REQUEST constant in src/mcp/types.py, also probably best to use "Invalid Request" message & rename helper accordingly